metric_providers/powermetrics/provider.py

```diff
@@ -181,7 +217,14 @@ def read_metrics(self, run_id, containers=None):
     def get_stderr(self):
         stderr = super().get_stderr()
 
-        if stderr is not None and str(stderr).find('proc_pid') != -1:
+        if stderr is not None and str(stderr).find('proc_pid') != -1 :
+            return None
+
+        # This has been showing up and we don't really understand why. Google has no results and looking at the
+        # strings of powermetrics doesn't show anything. There also seems to be no correlation with the interval.
+        # A shame we can't look into the code and figure this one out. For now we just ignore it as we don't really
+        # have any other chance to debug.
+        if stderr is not None and str(stderr).find('Second underflow occured') != -1 :
```

Comment:
Will that really do what we want? How I understand it you are returning here if the string "Second underflow" is in stderr.

However actually we want to return if ONLY this string is in stderr, not?

So we would have to iterate over all the lines and match?

Reply:
@ArneTR good point. Fixed please review